src/routes/management-api/users.ts

```
@@ -177,12 +177,18 @@ export class UsersMgmtController extends Controller {
       throw new HTTPException(400, { message: "Email is required" });
     }
 
+    if (user.connection !== "email") {
```

Ignasi has since changed the profile service to not send up `username-password`!

We were ignoring it anyway, but I'd rather throw an error

*if this is too aggressive* then we could start logging datadog errors instead? 

Ignasi and I feel that we don't actually know how many microservices are creating users one way or another :thinking: